frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -22,17 +22,21 @@ Feature: Applications managed by GitOps and Create secret
               And user navigates to Environments page
              Then user will see the list GitOps application groupings on the page
 
+        @regression @manual
+        Scenario: Application Details page for Applications: GO-01-TC04
+            Given user is on the Environments page
+              And user can see the Applications on the page
+             Then user can add new environments
 
         @regression @manual
         Scenario: Application Details page for Applications: GO-01-TC03
             Given user is on the Environments page
               And user can see the Applications on the page
-             When user clicks on the application
-             Then user can see the application Details page
+             Then user can see the columns "Application Name", "Git repository", "Environments" and "Last deployment"
```

Comment:
Based on their version of the GitOps operator, they will either see "Environments" (older; where all environments are in a comma-separated list) or "Environment Status" (newer; will have the # of environments for each status condition- Synced, OutOfSync, and Unknown)

Also, Application Name --> Application name (sentence case)

Reply:
@reginapizza Can you please share the support matrix. 

| GitOps Version  | OCP Version | Env Page  |
| ------------- | ------------- |  ------------- |
|   |   |   |
|   |   |   |

Reply:
If GitOps version has the dependency then i think it will be easy to handle in the test implementation code. May be i can rephrase the step as 

```Then user can see the columns attributes depending upon the "GioOps" version```

Reply:
working on confirming support matrix and I'll send that to you when I have it, but otherwise I think this statement is okay since it's generic.